suite-native/formatters/src/components/AccountAddressFormatter.tsx

```diff
@@ -4,18 +4,34 @@ import { useSelector } from 'react-redux';
 import { AccountsRootState, selectAccountLabel } from '@suite-common/wallet-core';
 import { Text, TextProps } from '@suite-native/atoms';
 import { AccountKey } from '@suite-common/wallet-types';
+import { prepareNativeStyle, useNativeStyles, mergeNativeStyleObjects } from '@trezor/styles';
 
 import { FormatterProps } from '../types';
 
 type AccountAddressFormatterProps = FormatterProps<AccountKey> & TextProps;
 
-export const AccountAddressFormatter = ({ value, ...rest }: AccountAddressFormatterProps) => {
+const addressStyle = prepareNativeStyle(_ => ({
+    // ellipsizeMode="middle" is not working  on Android with negative letterSpacing defined in @trezor/theme typography.
```

Comment:
maybe then use `extend` and Platform.OS === 'android' so we can keep letter spacing on iOS since it works here correctly

Reply:
fixed